src/Microsoft.AspNetCore.Mvc.Core/DisableRequestSizeLimitAttribute.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Microsoft.AspNetCore.Mvc.Core
```

Comment:
`Microsoft.AspNetCore.Mvc`

The general rule of thumb is that anything I user will have to type in a controller belongs in `Microsoft.AspNetCore.Mvc`